CONTRIBUTING.md

```diff
@@ -21,3 +21,131 @@ Be sure to include as much detail as possible including step-by-step description
 [new-issue]: https://github.com/codeforcauseorg/edu-client/issues/new
 [github-repo]: https://github.com/codeforcauseorg/edu-client/
 [issues-list]: https://github.com/codeforcauseorg/edu-client/issues
+
+## 📌 Setting up Locale Environment
+
+### 🚩 Forking repository :
+
+1. Firstly you have to make your copy of the project. For that, you have to fork the repository. You can find the fork button on the top-right side of the browser window. (Refer to the image below) 
+2. Kindly wait till it gets forked.
+3. After that copy will look like `<your-user-name>/edu-client` forked from `codeforcause/edu-client`.
+<img src="./doc/images/fork.png" alt="Demo fork image" align="center"/>
+
+### 🚩 Clone repository :
+
+1. Now you have your own copy of project. Here you have to start your work.
+2. Go to desired location on your computer where you want to set-up the project.
+3. Right click there and click on `git bash`. A terminal window will pop up
+4. Type the command `git clone <your-fork-url>.git` and hit enter.(Refer the image to copy url)
+5. Wait for few seconds till the project gets copied
+<img src="./doc/images/copy.png" alt="Demo copy image" align="center"/>
+
+### 🚩 Setting up remote :
+
+1. Now you have to set up remote repositories
+2. Type the command `git remote -v` this will list the remote connection to your repo.
+3. It will show something like this:
+```console
+
+origin  https://github.com/<your-user-name>/edu-client.git (fetch)
+origin  https://github.com/<your-user-name>/edu-client.git (push)
+
+```
+4. Now type the command `git remote add upstream https://github.com/codeforcauseorg/edu-client.git` this will set upstream as main directory
+5. Again type in command `git remote -v` to check if remote has been set up correctly
+6. It should show something like this :
+
+```console
+
+origin  https://github.com/<your-user-name>/edu-client.git (fetch)
+origin  https://github.com/<your-user-name>/edu-client.git (push)
+upstream        https://github.com/codeforcause/edu-client.git (fetch)
+upstream        https://github.com/codeforcause/edu-client.git (push)
+
+```
+
+### 🚩 Creating a branch :
+
+Whenever you want to contribute to any project. It is best practice that you should create a branch and push the branch as PR rather than directly pushing main/master branch
+
+1. `git branch` will list all the branched in the repository
+2. Now type `git branch <your-branch-name>`
+3. You can check the created branch by `git branch`
+4. But still if you start editing the edits will go to the main branch. To change the path of the edits type in `git checkout <your-branch-name>`
+5. Now you are ready to do the desired changes.
+
+## 📌 Setting up project using `npm` :
+
+1. Open this clonned folder in text editor of your choice
+2. If you want to use the project using `npm` then that comes along side when you download and install Node.js
+
+### 🚩 Running in Deployment mode :
+
```

Comment:
The new `npm install` command is added here in the new readme. Which is not a part of your readme

Reply:
alright i'll move this to the contributing section too